src/shared/fixed-periods/fixed-periods.js

```diff
@@ -76,8 +96,42 @@ const createUpdateFullPeriodTimeToDate = (multiplier) => {
     }
 }
 
+const monthlyCounts = {
+    [MONTHLY]: 1,
+    [BI_MONTHLY]: 2,
+    [QUARTERLY]: 3,
+    [SIX_MONTHLY]: 6,
+    [SIX_MONTHLY_APRIL]: 6,
+    [SIX_MONTHLY_NOV]: 6,
+}
+
+const monthlyPeriodTypes = Object.keys(monthlyCounts)
+
+const getPreviousMonthNumber = (startMonth, periodType) => {
+    const monthsToRemove = monthlyCounts[periodType]
+    if (startMonth - monthsToRemove < 0) {
+        return startMonth - monthsToRemove + 12
+    }
+    return startMonth - monthsToRemove
```

Comment:
calculation could be put in a variable

Reply:
After having read my comment again, I myself have no idea what I was trying to say here.. I'll simply resolve this

Reply:
I think you were suggesting to put `startMonth - monthsToRemove` calculation in a variable as it's used three times? I'm happy to update that, I just hadn't done any work/updates on data entry app for a while 🙂